src/tests/test_addresses.py

```diff
@@ -33,3 +33,21 @@ def test_encode(self):
         self.assertEqual(
             sample_address,
             addresses.encodeAddress(2, 1, unhexlify(sample_ripe)))
+
+    def test_base58(self):
+        """Check Base58 encoding and decoding"""
+        self.assertEqual(
```

Comment:
- these strings and numbers should be in `samples.py`
- to improve coverage, encode needs to test `0`, and decode needs to test an invalid character